tests/conftest.py

```diff
@@ -22,6 +24,23 @@
     "conda.testing.fixtures",
 )
 
+log = getLogger(__name__)
```

Comment:
You might want to use a different `__name__`, like `conda.tests` so it uses the same config as everything else.

Reply:
We got this for free by moving the fixture to `conda.testing`, see https://github.com/conda/conda/pull/13449/commits/e00239cc839d810a0b1ecfabea62743d82092224